jmeter/template.yaml

```
@@ -0,0 +1,89 @@
+Parameters: 
+  InstanceTypeParameter: 
+    Type: String
+    Default: t2.small
+    AllowedValues: 
+      - t2.small
+      - t2.medium
+  KeyNameParameter: 
+    Type: AWS::EC2::KeyPair::KeyName
+  SecurityGroupIdParamater:
+    Type: AWS::EC2::SecurityGroup::Id
+  SubnetIdParameter: 
+    Type: AWS::EC2::Subnet::Id
+  ImageIdParameter:
+    Type: AWS::EC2::Image::Id
+
+Resources:
+  Instance: 
+    Type: AWS::EC2::Instance
+    Metadata:
+      AWS::CloudFormation::Init:
+        configSets:
+            default: 
+              - install
+        install:
+          packages:
+            yum:
+              java-1.8.0-openjdk.x86_64: []
+              git.x86_64: []
+    Properties: 
+      LaunchTemplate:
+        LaunchTemplateId: !Ref TestTemplate
+        Version: !GetAtt TestTemplate.LatestVersionNumber
+      SubnetId: !Ref SubnetIdParameter
+      Tags: 
+      - Key: Name
+        Value: LoadTest
```

```suggestion
        - { Key: Name, Value: LoadTest }
        - { Key: CostCenter, Value: !Ref CostCenter }
        - { Key: Environment, Value: !Ref Environment }
        - { Key: Service, Value: !Ref Service }
        - { Key: Product, Value: !Ref Product }
        - { Key: Type, Value: !Ref Type }
        - { Key: Owner, Value: !Ref Owner }
```

We always set these tags for cost allocation, I also suggested the tags in the parameter section.